homeassistant/components/weather/strings.json

```diff
@@ -107,6 +125,24 @@
           "description": "[%key:component::weather::services::get_forecasts::fields::type::description%]"
         }
       }
+    },
+    "get_forecast_attr": {
+      "name": "Get forecast attribute values",
+      "description": "Get a single attribute within a time limit from the weather forecast.",
```

Comment:
limit is not really time limit any longer so perhaps rephrase to "get next x number of ..." something